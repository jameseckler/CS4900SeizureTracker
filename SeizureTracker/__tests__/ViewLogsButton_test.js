import React from 'react';
import renderer from 'react-test-renderer';
import ViewLogsButton from '../ClientScreens/screens/ClientHome/ViewLogsButton';

test ("View Logs Button renders properly", () => {
    
    const rendered = renderer.create(<ViewLogsButton click={()=> this.props.navigation.navigate('ViewLogs')} />).toJSON();
    expect(rendered).toBeTruthy();

});

it ("Test View Logs Button against snapshot", () => {

    const tree = renderer.create(<ViewLogsButton click={()=> this.props.navigation.navigate('ViewLogs')}/>).toJSON();
    expect(tree).toMatchSnapshot();

});