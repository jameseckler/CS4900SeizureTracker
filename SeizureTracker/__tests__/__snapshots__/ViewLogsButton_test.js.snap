// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test View Logs Button against snapshot 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "center",
      "backgroundColor": "transparent",
      "borderColor": "#E0E0E0",
      "borderRadius": 75,
      "borderWidth": 1,
      "justifyContent": "center",
      "marginVertical": 13.34,
      "opacity": 1,
      "paddingVertical": 15,
      "width": 637.5,
    }
  }
>
  <Text
    style={
      Object {
        "color": "white",
        "fontSize": 33.66831008530128,
        "fontWeight": "600",
        "paddingVertical": 13.34,
      }
    }
  >
    View Event Logs
  </Text>
</View>
`;
